controls/radgridview/events/column-resize-event.md

```diff
@@ -82,51 +82,51 @@ The **GridViewAutoGeneratingColumnEventArgs** class provides you with a few usef
 
 ## ColumnReordering
 
-The __ColumnReordering__ event occurs when RadGridView's columns are being reordered either programmatically or through the UI.
+A __ColumnReordering__ event occurs when you reorder RadGridView's columns programmatically or through the UI.
 
->The user can reorder columns only if the __CanUserReorderColumns__ is set to __True__ (which is the default value). 
+>You can reorder columns only if __CanUserReorderColumns__ is set to __True__ (which is the default value). 
 
-Through the __ColumnReorderingEventArgs__ you can access the following properties:
+Using __ColumnReorderingEventArgs__ you can access the following properties:
```

Comment:
**Regarding Comment 6:** Some properties available are read-only. Yes, you could set the **Cancel** property to True, but this does not mean you can set all properties. Some can be used for conditional statements. So I believe "Using" is most suitable here.